code/game/objects/items/devices/radio/radio.dm

```diff
@@ -503,28 +503,27 @@
 	. = ..()
 	set_frequency(FREQ_SYNDICATE)
 
-/obj/item/radio/borg/attackby(obj/item/W, mob/user, params)
-
-	if(W.tool_behaviour == TOOL_SCREWDRIVER)
-		if(keyslot)
-			for(var/ch_name in channels)
-				SSradio.remove_object(src, GLOB.radiochannels[ch_name])
-				secure_radio_connections[ch_name] = null
+/obj/item/radio/borg/screwdriver_act(mob/living/user, obj/item/tool)
+	if(!keyslot)
+		to_chat(user, span_warning("This radio doesn't have any encryption keys!"))
+		return
 
+	for(var/ch_name in channels)
+		SSradio.remove_object(src, GLOB.radiochannels[ch_name])
+		secure_radio_connections[ch_name] = null
 
-			if(keyslot)
-				var/turf/T = get_turf(user)
-				if(T)
-					keyslot.forceMove(T)
-					keyslot = null
+	if(keyslot)
+		var/turf/T = get_turf(user)
+		if(T)
+			keyslot.forceMove(T)
+			keyslot = null
 
-			recalculateChannels()
-			to_chat(user, span_notice("You pop out the encryption key in the radio."))
+	recalculateChannels()
+	to_chat(user, span_notice("You pop out the encryption key in the radio."))
 
-		else
-			to_chat(user, span_warning("This radio doesn't have any encryption keys!"))
+/obj/item/radio/borg/attackby(obj/item/W, mob/user, params)
```

Comment:
this too

Reply:
parent of this + parent of parent of this both had single letter variables which made me not want to replace it... but whatever i replaced it for all 3.

Reply:
some fixed are better than none, i don't think it matters if a child has a different has a different arg name (aside from consistency)